wordpress/wp-content/themes/canadian-digital-service/inc/forms.php

```diff
@@ -10,8 +10,8 @@ function my_field_container($field_container, $field, $form, $css_class, $style,
         if ($field->failed_validation) {
             $class .= ' form-group-error';
         }
-
-        return sprintf('<div id="%s" class="%s" style="%s">{FIELD_CONTENT}</div>', $field_id, $class, $style);
+// Change `div` for `li`
```

Comment:
do we need this inline comment? ideally we should just keep the code clean

Reply:
Sorry, I made this comment to refer to them at a Retro meeting and because it is still a WIP. They will be removed as soon as the patterns are stable.